.changeset/healthy-oranges-report.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro': patch
+---
+
+Throws an error if Astro.rewrite is used after the request body has been used
```

Comment:
```suggestion
Adds a new error `RewriteWithBodyUsed` that throws when `Astro.rewrite` is used after the request body has already been read.
```
Just a gentle massaging here! I know that I gave you the "present tense verb" advice, but I neglected to also give the helpful tip that we find these read well for the community when that verb completes the sentence, "This PR..."

e.g. "This PR adds.." , "This PR removes..." "This PR updates..." (So, the PR itself is the unexpressed subject of the sentence.)

And again, I'm being overly thorough here just to model some ideals. It's not *criticial* to include the name of the new error, but in general, whenever any new `thing` is added to the codebase, I like to have it mentioned in the changeset. For an error, ok maybe overkill. 😂 But for PRs that "Adds a new config option `option`" I find I am often coming in here saying, "Let's mention the actual thing that was added!" So I'm just modelling that here! 😄

Reply:
Thanks! This is super helpful. I always forget the required for and have to check. I wonder if there's a way to update the actions message to remind about it.